libs/ngx-dsfr/radio/src/lib/radio.module.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Angular imports
+ */
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { FormsModule } from '@angular/forms';
+
+/**
+ * Internal imports
+ */
+import { DsfrRadioComponent } from './radio.component';
+import { TestHostComponent } from './testing/test-host.component';
+
+@NgModule({
+  imports: [CommonModule, FormsModule],
```

Comment:
If you manage to make it work without `ngModel` in the Template, normally you'd be able to also remove the import of `FormsModule`